### PR TITLE
[WIP]Clarify that UOW can't access message context in v6

### PR DESCRIPTION
**Pull request by andreasohlund**
Unsure where in the upgrade guide to put a note?

@Particular/docs-maintainers can I use included in the samples like I do?

### CODE REVIEW

File: nservicebus/uow-access-to-context.include.md [hunk 1]
```diff
@@ -0,0 +1 @@
+NOTE: In NServiceBus Versions 6 `IManageUnitsOfWorks` doesn't have access to the incoming message context. If access to headers and/or message body is needed implemement a [unit of work using behaviors instead](samples/pipeline/unit-of-work).
```
Comment (SimonCropp) [inline nservicebus/uow-access-to-context.include.md]: "Versions 6 and above"  

and i think "NServiceBus " is redundant in this context?

Comment (SimonCropp) [inline nservicebus/uow-access-to-context.include.md]: samples/pipeline/unit-of-work will be a broken link

Comment (andreasohlund) [inline nservicebus/uow-access-to-context.include.md]: So `/samples/pipeline/unit-of-work/sample.md` ?

**Comment by andreasohlund** [comment]
@SimonCropp ideas on where to put the note in the upgrade guide, didn't find a good place and creating a new sup paged seemed YAGNI?

**Comment by andreasohlund** [comment]
Thanks Simon, extensions seems like a good place for the note.

**Pull request merged**